pkg/pagerduty/service.go

```diff
@@ -453,6 +453,26 @@ func parseIncidentNumbers(incidents []pdApi.Incident) []uint {
 	return incidentNumbers
 }
 
+// generateServiceName checks if FedRamp is enabled. If it is, it returns
+// an anonymized PD service name.
+func generatePDServiceName(data *Data) string {
+	if config.IsFedramp() {
+		return data.ClusterID
```

Comment:
```suggestion
		return data.ServicePrefix + "-" + data.ClusterID
```

Any reason not to prefix?  If we don't do this the name is not unique if we have more than one PDI associated with a cluster.

Reply:
The general guidance for FedRAMP alerting has been to provide as little information as possible in these alerts while still providing a way to identify the cluster so that we can access the alertmanager instance directly for additional information. But the possibility of multiple PDIs hadn't occurred to me. I'll add back the prefix.